torchvision/transforms/v2/functional/_meta.py

```diff
@@ -214,6 +214,11 @@ def convert_bounding_box_format(
     if not torch.jit.is_scripting():
         _log_api_usage_once(convert_bounding_box_format)
 
+    if isinstance(old_format, str):
```

Comment:
I didn't update the type annotations of `convert_bounding_box_format` because it would make mypy unhappy. If it fails, I'll silence it. I think it means that torchscript won't support passing strings as the format, but who cares